docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -97,13 +97,13 @@ Date and time functions
     zone, into a ``timestamp(9) with time zone``. The time defaults to
     ``00:00:00.000000000``, and the time zone defaults to the session time zone::
 
-        SELECT from_iso8601_timestamp('2020-05-11');
+        SELECT from_iso8601_timestamp_nanos('2020-05-11');
```

Comment:
This does not actually work despite the docs about the default to zero above. I get 

Text '2020-05-11' could not be parsed at index 10

Reply:
Using cli 357 connected trino 357, from_iso8601_timestamp_nanos() is not recognized. Is it supposed to be only in SEP?

Reply:
I tested this locally with Trino 356.